### DISCUSSION TITLE
Sunny Portal / keine Geräte hinzufügen.

### DISCUSSION THREAD

**Discussion by dennisbausch** [camueller/SmartApplianceEnabler#336] [Allgemein]
Hallo zusammen , nach dem ich nach dem neu aufsetzen ein den import der alten Konfiguration Dateien alle Geräte wieder im SAE hatte, habe ich festgestellt das 2 nicht richtig laufen !  

Hab ein wenig probiert bin dann aber nicht weiter gekommen und habe insgesamt 7 Geräte im Sunny Portal und SAE gelöscht und neu im SAE angelegt.  Nur komme ich jetzt nicht mehr weiter , es werden sowohl die alten als auch die neuen Geräte angezeigt wenn ich nach Geräten suche und ich bekomme keins mehr Angelegt. 

Ich habe schon ein paar tage gewartet in der Hoffnung das die alten gerate wider verschwinden aber leider passiert nichts.

<img width="782" alt="Bildschirm­foto 2022-09-10 um 14 08 51" src="https://user-images.githubusercontent.com/59669612/189482671-a3aca47a-f8eb-4e6c-81f6-a3729552649a.png">

**Comment by dennisbausch** [comment]
Hat jemand eine Idee woran das liegen kann ?

anbei ein gekürzter Log:
[rolling-2022-09-13.log](https://github.com/camueller/SmartApplianceEnabler/files/9558856/rolling-2022-09-13.log)

**Reply by zeron850** [reply]
Hallo, 

hat der Tipp mit dem Reset bei dir funktioniert? Ich habe das gleiche Problem wie du... 

Danke

**Comment by camueller** [comment]
Du könntest den Home Manager mal resetten. Beim HM 2.0 dazu die Reset-Taste länger als 6s gedrück halten.

**Comment by dennisbausch** [comment]
Der Reset hat leider nichts gebracht … daraufhin habe ich den Support angeschrieben und folgende Info bekommen :

> 
> 
> Bitte beachten Sie, dass der Home Manager zwar bis zu 24 Geräte erfassen kann, davon jedoch nur 12 als Verbraucher im aktiven Energiemanagement. 
>  
> Da Sie bereits 12 Verbraucher + den EV Charger erfasst haben, ist eine Erfassung weiterer Verbraucher nicht möglich.
> 



es gab doch mal einen Ansatz das zu umgehen ? 

Muss ich erst 2 Verbraucher löschen um da ok einen Schwung 11 wieder hinzuzufügen ? 

Danke

**Reply by camueller** [reply]
Ja. Steht auch so in der Dokumentation.
